### PR TITLE
localize 'Containers'

### LINKED ISSUE #578: [Spanish] localize 'Containers'
Localize the `content/en/container.md` document to Spanish

### PR DESCRIPTION
**Pull request by josefloressv**
### Describe your changes
Localize the `content/en/container.md` document to Spanish
See more comments from a previously closed PR https://github.com/cncf/glossary/pull/1740 (Include the @krol3 suggestions)

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #578

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by krol3** [comment]
@josefloressv  esta faltando el DCO, `Signed-off-by: Author Name <authoremail@example.com>`

**Comment by josefloressv** [comment]
Gracias @krol3, corregido!

File: content/es/container.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Los archivos disponibles del contenedor se empaquetan como una imagen de contenedor. 
+Los contenedores se ejecutan uno al lado del otro en la misma máquina,
+pero, por lo general, el sistema operativo evita que los procesos de contenedores independientes interactúen entre sí.
+
+## Problema que aborda
+
+Antes de que los contenedores estuvieran disponibles, se necesitaban máquinas separadas para ejecutar aplicaciones.
+Cada máquina requeriría su propio sistema operativo, que requiere CPU, memoria y espacio en disco,
+todo para que una aplicación individual funcione.
+Además, el mantenimiento, la actualización y el inicio de un sistema operativo es otra fuente importante de trabajo.
```
Comment (electrocucaracha) [inline content/es/container.md:20]: No estoy muy seguro de que pueda ser una opción. 
```suggestion
Además, el mantenimiento, la actualización y el inicio de un sistema operativo es considerablemente otra fuente de esfuerzo.
```
\cc @ramrodo @krol3 @92nqb

**Review by electrocucaracha — APPROVED** [review]
Gracias por corregir los problemas en el CI. Solo tengo una duda respecto a un renglón que me gustaría consultar con los demás

File: content/es/container.md (hunk 1 above)
Comment (ramrodo) [reply content/es/container.md:20]: Ambas me parecen correctas, sin embargo, la sugerencia de @electrocucaracha me parece más adecuada y se entiende mejor.

**Review by ramrodo — APPROVED** [review]
/lgtm

**Pull request merged**